node/image-classification-with-huggingface/README.md

```diff
@@ -0,0 +1,132 @@
+# Image Classification with Hugging Face
+
+This function uses the Hugging Face API to classify images. It takes an image file from Appwrite storage and sends it to the Hugging Face API for classification. The API returns a list of labels and their scores and records it in the database. This function also supports receiving file events from Appwrite Storage.
+
+## 🧰 Usage
+
+### POST /
+
+**Parameters**
+| Name | Description | Location | Type | Sample Value |
+|------------|-------------|----------|--------|--------------|
+| imageId | Appwrite File ID of Image | Body | String | `65c6319c5f34dc9638ec` |
+
+This function also accepts body of a file event from Appwrite Storage.
+
+**Response**
+
+Sample `200` Response:
+
+Image of a dog on Appwrite Storage is sent as input and is recognized.
+
+```json
+[
+  {
+    "label": "Weimaraner",
+    "score": 0.9862596988677979
+  },
+  {
+    "label": "German short-haired pointer",
+    "score": 0.005923726130276918
+  },
+  {
+    "label": "Chesapeake Bay retriever",
+    "score": 0.0009203946683555841
+  },
+  {
+    "label": "vizsla, Hungarian pointer",
+    "score": 0.0003758686943911016
+  },
+  {
+    "label": "Rhodesian ridgeback",
+    "score": 0.0003360954870004207
+  }
+]
+```
+
+Sample `404` Response:
+
+```json
+{
+  "error": "Image not found"
+}
+```
+
+## ⚙️ Configuration
+
+| Setting           | Value                      |
+| ----------------- | -------------------------- |
+| Runtime           | Node (18.0)                |
+| Entrypoint        | `src/main.js`              |
+| Build Commands    | `npm install`              |
+| Permissions       | `any`                      |
+| Timeout (Seconds) | 15                         |
+| Events            | `buckets.*.files.*.create` |
+
+## 🏗️ Setup
+This function has a script that will setup the databases, collections and storage buckets for you, you just need to add the environment variables to the `.env` file, run `npm install` and finally run the following command:
+```
+npm run setup
+```
+
+You do not need anything to be created before running the setup script, it will create everything for you.
```

Comment:
@PineappleIOnic The setup part needs to be in the configuration section like the existing examples
https://github.com/appwrite/templates/tree/main/node/payments-with-stripe#%EF%B8%8F-configuration